gradle.properties

```diff
@@ -12,7 +12,7 @@ artifactUrl = 'https://github.com/Unidata/unidata-jekyll-gradle'
 
 # sets the version of the Unidata theme and plugin set that is shipped with the Unidata jekyll plugin
 unidataThemeVersion=0.0.3
```

Comment:
Does the `unidataThemeVersion` need bumping?

Reply:
I didn't make any changes to the theme except the version of the plugins that it depends on, so I'm pretty sure no...but this also not urgent now that netcdf-java is passing again, so it can wait for a review from @lesserwhirls if needed.

Reply:
Correct @haileyajohnson - the Jekyll theme, the Jekyll plug-in, and the gradle plug-in are independently versioned. In this case, we should just need to bump the Jekyll plug-in version, keep the theme version as is, and make a new release of the gradle plug-in.